test/Interop/Cxx/operators/member-inline-module-interface.swift

```diff
@@ -15,3 +15,25 @@
 
 // CHECK: struct HasDeletedOperator {
 // CHECK: }
+
+// CHECK: struct ReadWriteIntArray {
+// CHECK:   subscript(x: Int32) -> Int32 { mutating get set }
+// CHECK: }
+
+// CHECK: struct ReadOnlyIntArray {
+// CHECK:   subscript(x: Int32) -> Int32 { mutating get }
```

Comment:
I guess they can be considered an implementation detail rather than a part of the public interface, but yeah, it doesn't hurt to check their presence as well them. Fixed.